content/asset-tracking/index.textile

```diff
@@ -0,0 +1,165 @@
+
+- asset (trackable) := Item that the customer wants to track such as driver, vehicle, container, and so on. This also known as a @Trackable@ in the SDKs.
+- driver := Person driving the asset physically. Could also be a machine moving the asset under software direction, such as in the case of a delivery drone.
+- customer := Entity that uses the Ably Asset Tracking SDKs to build their product to solve the asset tracking problem for their users.
+- end user := Person that uses the product built by the customer and uses the asset tracking capabilities.
+
+Each asset being tracked must have a @trackableId@, and this must be globally unique within an Ably asset tracking account.
+
+The Ably Asset Tracking SDKs also allow for various battery optimization scenarios, so frequency of updates can be modified based on various parameters such as distance to destination, subscriber preference, map zoom level, and current battery levels.
+
+h2(#battery-optimization). Battery optimization scenarios
+
+One of the biggest challenges in asset tracking is combining high resolution of updates with low battery consumption on the asset tracking device. For example, in a delivery tracking use case you may want your users to have a smooth and realtime visualization of where their delivery is on the map, but at the same time sending locations from the driver's device frequently will rapidly drain its battery.
+
+There are two key factors that impact asset tracking device battery usage:
+
+1. GPS sensor - obtaining location of the asset.
+2. Networking - sending location updates over network.
+
+It is hard to reduce the battery consumption of the GPS sensor in some situations, such as when the device is being used for navigation at the same time as tracking an asset. However, it is possible to adjust the frequency of updates.
+
+The following scenarios are examples of how update resolution can be changed depending on the state of the asset and subscribers:
+
+* Updates can be sent to the subscriber more often when asset is closer to the destination.
+* Updates don't need to occur as frequently when the user of the subscribing SDK is not observing the location of the asset. The frequency of updates can be reduced, for example, if the user of delivery app is not checking the app for the driver's location.
+* Different subscribers may have different preferences regarding the resolution of location updates.
+* When the driver's battery is running low, location updates can be transmitted at lower frequency.
+
+Using the approaches similar to these, an optimal resolution of updates can be identified, which provides a good user experience when it is needed, such as when the driver is close to the destination, and the user is observing the location of the driver in the app. This approach also reduces battery usage when possible, for example when the driver is far from the destination and the user is not observing location updates.
```

Comment:
I'm not too sure what "Using _the_ approaches similar to these" is supposed to mean in this context. I'm wondering if this would be better written as either:

- "Using these approaches"
- "Using approaches similar to these"